### ISSUE TITLE
Training logs

### ISSUE THREAD

**Issue by maning** [maning/robosat-viz#1]
Summary of training logs:

date | training sample | mean validation Iou | time | notes | chart
-- | -- | -- | -- | -- | --
2018-07-01 | training - 6400, validation - 1589 | Validate loss: 0.1794, mean IoU: 0.6323 | real    319m29.377s | no hard negatives | ![history-00010-of-00010](https://user-images.githubusercontent.com/353700/43052860-eb1c87ca-8e46-11e8-99ff-c6c5bd9ade9c.png)
2018-07-07 | training - 9698, validation - 1212, evaluation - 1213 | Validate loss: 0.0397, mean IoU: 0.7999 | real    266m35.701s | cleaned TP tiles, with hard negatives, new resnet encoder/decoder, batch size - 4, epoch 10 | ![history-00010-of-00010](https://user-images.githubusercontent.com/353700/43052902-36fd5dc2-8e47-11e8-8d29-53c1f4a2fe85.png)
2018-07-14 | training - 12185, validation - 1524, evaluation - 1522 | Validate loss: 0.0372, mean IoU: 0.8133 | real    329m35.657s | cleaner TPs, more hard negatives, positives - 7104 (46%), negatives - 8,133 (54%), batch size - 4, epoch 10 | ![history-00010-of-00010](https://user-images.githubusercontent.com/353700/43052923-6077b6de-8e47-11e8-9cf2-335f09f868ac.png)
2018-17-18 | same as ^^ | Validate loss: ??, mean IoU: 0.82xx | did not log | same as ^^batch size - 4, epoch 50 | ![history-00050-of-00050](https://user-images.githubusercontent.com/353700/43052935-7bb21c28-8e47-11e8-8983-a79dad49aad8.png)
2019-06-11 |training - 22,921, validation - 5,248, evaluation - 3,853 | Validate loss: 0.0120, mIoU: 0.861, building IoU: 0.735, MCC: 0.841 |1895m6.832s | | ![history-00049-of-00050](https://user-images.githubusercontent.com/353700/60005573-5ca92200-96a1-11e9-8809-71dc2a20e60c.png) 
2019-06-13 | same as above |Validate loss: 0.0013, mIoU: 0.876, building IoU: 0.765, MCC: 0.861 |1584m29.546s | | ![history-00100-of-00100](https://user-images.githubusercontent.com/353700/60005600-6763b700-96a1-11e9-9cbd-099235df6279.png)

**Comment by maning** [comment]
Training data sources:

name | image | date | resolution | sensor | # of tiles | positive:negative ratio (%)| user
|---|---|---|---|---|---|---|---|
previous training data | ![985250](https://user-images.githubusercontent.com/353700/60009452-7e59d780-96a8-11e9-9935-2489259215f9.png)| May-2017 | 2-3 cm | DJI Mavic Pro | 15,231 | 46%:54% | Dan Joseph
[Arteche, Eastern Samar]( https://map.openaerialmap.org/#/125.12054443359374,11.962753823592042,9/square/13231223003212/59e62beb3d6412ef7220c5c1?resolution=high&_k=2rm9b6) | ![976540](https://user-images.githubusercontent.com/353700/60009886-3b4c3400-96a9-11e9-873f-de7125ff8903.png) | 2014-12-01 | 4 cm | N/A | 2,256 | 12%:88% |SkyEye  
[Bacong River, Culasi, Antique](https://map.openaerialmap.org/#/122.06859827041625,11.409255248783925,15/square/1323033233021211/59e62b933d6412ef7220a248?_k=il7ge3) | ![981683](https://user-images.githubusercontent.com/353700/60010845-325c6200-96ab-11e9-9048-b552da268a43.png) | 2017-05-24| 3 cm | DJI Mavic Pro | 5,895 |0.8%:99.2% | Dan Joseph
[Malalison Island, Culasi, Antique](https://map.openaerialmap.org/#/122.02000737190245,11.413893131178812,16/square/132303323302030102/59e62b943d6412ef7220a2bc?_k=2a1nm2) | ![981659](https://user-images.githubusercontent.com/353700/60011476-6e43f700-96ac-11e9-9ff4-1da86c28dad3.png) | 2017-05-23| 4 cm | DJI Mavic Pro | 2,650 | 2%:98%| Dan Joseph
[Narvacan Sulvec Port Road](https://map.openaerialmap.org/#/120.45186728239061,17.447123369400124,18/user/5b44040e2b6a08001185f952/5b443a472b6a08001185f96a?resolution=high&_k=h0azhq) | ![945349](https://user-images.githubusercontent.com/353700/60011934-6cc6fe80-96ad-11e9-952c-8e814bf74a89.png) | 2018-03-14 | 4 cm | DJI Phantom 4 | 773 | 8%:92%|  Leonard G. Soriano & Ian Gongona
[Sugarcane LAREC Mill District, Pampanga](https://map.openaerialmap.org/#/120.5328915,14.98907255286197,16/square/13230303103/5b518afc4bda9e648eedbd9d?resolution=high&_k=kurjqg)|![960254](https://user-images.githubusercontent.com/353700/60012398-61280780-96ae-11e9-8ba1-e99011753890.png) |2017-03-30 | 5 cm | EB Sensefly | 2,748 | 4%:96% | Michael Andrew Manalili
